book/src/external-resizer.md

```diff
@@ -0,0 +1,25 @@
+# CSI external-resizer
+
+## Status and Releases
+
+**Git Repository:** https://github.com/kubernetes-csi/external-resizer/
+
+**Status:** Alpha
+
+Latests release | Branch | Compatible with CSI Version | Container Image | Min k8s Version | Max k8s version
```

Comment:
Make sure to incorporate the min/max CSI version change that @msau42 is working on for the other sidecars in to this PR as well.